packages/styles/__tests__/styles-test.js

```diff
@@ -80,3 +81,22 @@ describe('@carbon/styles', () => {
     });
   });
 });
+
+describe('type custom properties', () => {
+  it('should emit the css Custom properties for non-fluid typography', async () => {
+    const { result } = await render(`
+        @use '../scss/type' as *;
+
+        .selector {
+          @include emit-type-custom-properties()
+        }
+      `);
+    const { stylesheet } = css.parse(result.css.toString());
+    const rules = stylesheet.rules.filter(
+      (rule) => rule.selectors[0] === '.selector'
+    );
+    const declarations = rules[0].declarations;
+
+    expect(declarations.length).toBe(191);
+  });
+});
```

Comment:
I just realized... I think the proper location for this to live would be in `packages/styles/scss/__tests__/type-test.js`